dbt_subprojects/dex/macros/models/_project/uniswap_compatible_trades.sql

```diff
@@ -118,4 +118,72 @@ SELECT
     , dexs.evt_index
 FROM
     dexs
+{% endmacro %}
+
+{% macro uniswap_compatible_v4_trades(
+    blockchain = null
+    , project = 'uniswap'
+    , version = '4'
+    , Pair_evt_Swap = source('uniswap_v4_sepolia', 'PoolManager_evt_Swap') 
+    , Factory_evt_PoolCreated = source('uniswap_v4_sepolia', 'PoolManager_evt_Initialize')
```

Comment:
```suggestion
    , Pair_evt_Swap = null
    , Factory_evt_PoolCreated = null
```

Reply:
lets default to null, so the job would fail if a model doesn't supply the proper source params to pass in.
we should expect all models to provide this value, as it's one model per chain, which will change the schema name.

Reply:
Maybe for v4:
```suggestion
    , PoolManager_evt_Swap = null
    , PoolManager_evt_Initialize = null
```

Reply:
updated